testing.sagebrush.master/Stepwat_Inputs/Input/species.in

```diff
@@ -75,16 +75,16 @@
 
 # name   rg  onoff  age  slow  disturb  eind  vegindv  tclass  clonal   irate   ratep   pestab   minbio    maxbio  cosurv
   artr   1     1     100   4      4       1      0        2   	  n		0.35    0.900    0.15    2.0       1000.0   0.01
-  trdu   2     0      1    2      3       5      0        2       n     0.947   0.900    0.03    0.035      4.700   0.00
-  cryp   2     1      1    2      3      10      0        2       n     0.947   0.900    0.20    0.035      4.700   0.00
-  amre   3     0      1    2      3       5      0        2       n     0.947   0.900    0.03    0.035      4.429   0.00
-  chen   3     1      1    2      2       8      0        1       n     0.947   0.900    0.20    0.035      4.429   0.00
-  acmi   4     0     10    2      3       6      0        2       n     0.426   0.900    0.15    0.035      5.000   0.01
-  phho   4     1     10    2      3      15      0        2		  n     0.426   0.900  	 0.15    0.035   	22.00   0.01
-  arfr   5     1      5    2      3       5      0        2       n     0.426   0.900    0.10    0.035      7.480   0.01
-  lipu   5     0     35    2      3       4      0        2       n     0.426   0.900    0.10    0.035      7.480   0.01
-  brte   6     1      1    2      3      20      0        2       n     0.947   0.900    0.15    0.020      5.000   0.00
-  vuoc   6     0      1    2      1      16      0        2       n     0.947   0.900    0.03    0.020      5.000   0.00
+  trdu   2     0      1    2      3       5      0        2       n     0.947   0.900    0.03    0.5      4.700   0.00
+  cryp   2     1      1    2      3      10      0        2       n     0.947   0.900    0.20    0.5      4.700   0.00
+  amre   3     0      1    2      3       5      0        2       n     0.947   0.900    0.03    0.5      4.429   0.00
+  chen   3     1      1    2      2       8      0        1       n     0.947   0.900    0.20    0.5      4.429   0.00
+  acmi   4     0     10    2      3       6      0        2       n     0.426   0.900    0.15    0.5      5.000   0.01
+  phho   4     1     10    2      3      15      0        2		  n     0.426   0.900  	 0.15    0.5   	22.00   0.01
+  arfr   5     1      5    2      3       5      0        2       n     0.426   0.900    0.10    0.5      7.480   0.01
+  lipu   5     0     35    2      3       4      0        2       n     0.426   0.900    0.10    0.5      7.480   0.01
+  brte   6     1      1    2      3      20      0        2       n     0.947   0.900    0.15    0.5      5.000   0.00
+  vuoc   6     0      1    2      1      16      0        2       n     0.947   0.900    0.03    0.50      5.000   0.00

```

Comment:
these values were pushed for testing purposes, but they need to be reverted before merging this branch into master.

Reply:
resolved by https://github.com/DrylandEcology/STEPWAT2/pull/481/commits/6b2e3c5f6c05eadaebeaa70ea2f6836cc0f9b3d2